### PR TITLE
Filelist state

### PR DESCRIPTION
**Pull request by YourLittleHelper**
Incomplete, but it's ok. There is enough to start implementing design.

### CODE REVIEW

File: HugoCatFront.iml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (IntarsPreticis) [inline HugoCatFront.iml]: Not sure what this file is for. But fast google search makes me think this should be in gitignore?

File: public/index.html [hunk 2]
```diff
@@ -5,11 +5,11 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width,initial-scale=1.0">
     <link rel="icon" href="<%= BASE_URL %>favicon.ico">
-    <title>hugo-cat-front</title>
+    <title>hugo-typescript</title>
```
Comment (IntarsPreticis) [inline public/index.html]: Does the project title really have to  describe the build technologies of it?

File: src/file-manager/MT.ts [hunk 3]
```diff
@@ -0,0 +1,4 @@
+export class MT {
```
Comment (IntarsPreticis) [inline src/file-manager/MT.ts]: I would add a comment what does MT stands for. I have no idea even looking further into code.. or should I just know it? :)

File: public/index.html [hunk 4]
```diff
@@ -4,8 +4,6 @@
     <meta charset="utf-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width,initial-scale=1.0">
-    <link rel="icon" href="<%= BASE_URL %>favicon.ico">
-    <title>hugo-typescript</title>
```
Comment (IntarsPreticis) [inline public/index.html]: I dont think <title> tag should be removed, just give it a correct description. Plus w3schoold states that: "If you omit the <title> tag, the document will not validate as HTML."

Comment (YourLittleHelper) [reply public/index.html]: Tilde includes only body and css/scripts inside their own web page, our index file is only for dev purposes. But yeah, probably better to set title for development too.

**Pull request merged**